great_ape_safe/great_ape_safe.py

```diff
@@ -165,6 +170,8 @@ def take_snapshot(self, tokens):
 
 
     def print_snapshot(self, csv_destination=None):
+        if 'init_snapshot' in self.__dir__():
+            return
```

Comment:
also note: `print_snapshot` will trigger here and cause exception, added check prevent great ape safe getting confused